cleanlab/object_detection/rank.py

```diff
@@ -0,0 +1,1049 @@
+# Copyright (C) 2017-2023  Cleanlab Inc.
+# This file is part of cleanlab.
+#
+# cleanlab is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# cleanlab is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with cleanlab.  If not, see <https://www.gnu.org/licenses/>.
+
+"""Methods to rank and score images in an object detection dataset (object detection data), based on how likely they
+are to contain label errors. """
+
+import warnings
+from typing import List, Dict, Any, Optional, Tuple, TypedDict
+import numpy as np
+import copy
+
+AuxiliaryTypesDict = TypedDict(
+    "AuxiliaryTypesDict",
+    {
+        "pred_labels": np.ndarray,
+        "pred_label_probs": np.ndarray,
+        "pred_bboxes": np.ndarray,
+        "lab_labels": np.ndarray,
+        "lab_bboxes": np.ndarray,
+        "similarity_matrix": np.ndarray,
+        "min_possible_similarity": float,
+    },
+)
+
+
+def get_label_quality_scores(
```

Comment:
For some of the docstrings, see my comments above for find_label_issues docstring. Make sure you incorporate the best/relevant parts of the find_label_issues docstring into this one.